docs/autodiff/tensor.html

```diff
@@ -153,20 +153,22 @@ <h1 class="title">Module <code>nujo.autodiff.tensor</code></h1>
     def add_grad_dependency(self, wrt: &#39;Tensor&#39;, weight: ndarray) -&gt; None:
         self._grad_dependencies.append((wrt, weight))
 
-    def compute_grad(self, debug=False) -&gt; None:
```

Comment:
As i see there is code duplication. Think if you coul avoid it

Reply:
Do you mean that the `_compute_grad` function is duplicated? If so, it is duplicated only in the docs, because of the way they are generated. Is this fine, or should we address it anyway?